app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -11,7 +10,7 @@
 
 @pytest.mark.nondestructive
 def test_create_new_experiment(selenium, base_url):
-    experiment_name = f"name here remote {random.randint(0, 100)}"
+    experiment_name = "test_create_new_experiment"
```

Comment:
Oh and I changed the names from random to the name of the test because I couldn't figure out which experiment was created by which test when inspecting in VNC

Reply:
Lovin it! And seems like potentially fewer risks of an experiment naming collision if `random.randomint` ever happened to return the same 0..100 value for different tests in a run.

Does Python/Pytest have a way to get the current function's name from within a function?
I found https://stackoverflow.com/questions/17726954/py-test-how-to-get-the-current-tests-name-from-the-setup-method; but maybe not worth the hassle.

Reply:
Nah no need to be clever about it.